lib/parser.js

```diff
@@ -0,0 +1,43 @@
+var fs = require('fs');
+
+var parser = module.exports = function(path, callback) {
+  fs.readFile(path, 'utf-8', function(err, file) {
+    file = file.split('\n');
+    var course = {};
+    for(var i = 0, j = 1; i < file.length; i++) {
```

Comment:
what is j for? if you're going to have a second counter make it one that makes sense.